modules/monitoring-enabling-an-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    alertmanager: true <1>
+----
+<1> Set the `alertmanager` key to `true` to enable a dedicated instance of the Alertmanager for user-defined projects in a cluster. Set the value to `false` or omit the key entirely to disable the Alertmanager for user-defined projects.
++
+. Save the file to apply the changes. The dedicated instance of Alertmanager for user-defined projects starts automatically.
+
+.Verification
+
+. Verify that the `alertmanager-user-workload` pods have been created:
++
+[source,terminal]
+----
+# oc -n openshift-user-workload-monitoring get pod
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                   READY   STATUS    RESTARTS        AGE
+alertmanager-user-workload-0           6/6     Running   0               6m27s
+alertmanager-user-workload-1           6/6     Running   0               6m27s
+prometheus-operator-549fc76b94-s9kgp   2/2     Running   0               71m
+prometheus-user-workload-0             6/6     Running   0               71m
+prometheus-user-workload-1             6/6     Running   0               71m
+thanos-ruler-user-workload-0           3/3     Running   1 (6m20s ago)   6m25s
+thanos-ruler-user-workload-1           3/3     Running   1 (6m25s ago)   6m29s
+----
++
+. Verify that the Alertmanager instance has started:
++
+[source,terminal]
+----
+#  oc -n openshift-user-workload-monitoring get alertmanager
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME            VERSION   REPLICAS   AGE
+user-workload   0.24.0    2          100s
+----
++
+. Verify that the Alertmanager service has started:
++
+[source,terminal]
+----
+#  oc -n openshift-user-workload-monitoring get svc
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                       AGE
+alertmanager-operated                     ClusterIP   None             <none>        9093/TCP,9094/TCP,9094/UDP    113s
+alertmanager-user-workload                ClusterIP   172.30.249.4     <none>        9095/TCP,9092/TCP,9097/TCP    118s
+prometheus-operated                       ClusterIP   None             <none>        9090/TCP,10901/TCP            111m
+prometheus-operator                       ClusterIP   None             <none>        8443/TCP                      112m
+prometheus-user-workload                  ClusterIP   172.30.176.215   <none>        9091/TCP,9092/TCP,10902/TCP   112m
+prometheus-user-workload-thanos-sidecar   ClusterIP   None             <none>        10902/TCP                     112m
+thanos-ruler                              ClusterIP   172.30.76.238    <none>        9091/TCP,10901/TCP            112m
+thanos-ruler-operated                     ClusterIP   None             <none>        10902/TCP,10901/TCP           111m
+----
```

Comment:
I am kindly think the it only show how to enable a dedicated instance of Alertmanager, the verification steps does not show how to prove the dedicated Alertmanager works
1. if enabled the dedicated Alertmanager, the user projects alerts are sent to this alertmanager instance only
2. if disabled the dedicated Alertmanager, the user projects alerts are sent to platform alertmanager(under openshift-monitoring)

Reply:
I think that the "oc -n openshift-user-workload-monitoring get alertmanager" step should be enough as it will prove that the user didn't mess up the edition of the configmap.
If the Alertmanager resource is created but the Alertmanager pods can't be started for some reason (no node capacity, failed scheduling, ...), there would be alerts firing.

Reply:
I've deleted all of the verification steps except for `oc -n openshift-user-workload-monitoring get alertmanager`